src/components/themed/SideMenu.tsx

```diff
@@ -162,21 +162,21 @@ export function SideMenuComponent(props: Props) {
     navigation.navigate('edgeAppStack', { screen: 'coinRanking' })
   }
 
-  const handleShareApp = () => {
+  const handleShareApp = async () => {
     // Generate anonymized referral ID
     const data = Uint8Array.from(Buffer.from(account.rootLoginId, 'hex'))
     const refId = hashjs.sha256().update(data).digest('hex').replace('0x', '').substring(0, 10)
 
     const url = `${config.website}?af=appreferred_${refId}`
-
+    const subject = sprintf(lstrings.share_subject, config.appName)
     const shareOptions = {
       failOnCancel: false,
-      title: lstrings.share_subject,
-      subject: lstrings.share_subject,
+      title: subject,
+      subject,
       message: lstrings.share_message,
       url
     }
-    Share.open(shareOptions).catch(e => showError(e))
+    await Share.open(shareOptions).catch(e => showError(e))
```

Comment:
Remove the `.catch` - the `EdgeTouchableOpacity` deals with that as part of the debounce.